### PR TITLE
Fix lua-language-server annotations and typo's

### PR DESCRIPTION
**Pull request by bananasov**
This PR makes the annotations used follow the "standard" more and fixes 2 spelling mistakes that i could find. I might have missed more in the 400+ of lines.

### CODE REVIEW

File: src/kristly.lua [hunk 1]
```diff
@@ -7,22 +7,22 @@ local kristlyWS = { ws = nil }
 --                                 UTILS                                        --
 ----------------------------------------------------------------------------------
 
---- A basic JSON Post function to the krist api.
--- Decodes the response from json into a table.
---- @param endpoint string The json endpoint. The first part of the url should not be included.
---- @param body string A stringified body
---- @return table result The response from the krist API
+---A basic JSON Post function to the krist api.
+---Decodes the response from json into a table.
```
Comment (piprett) [inline src/kristly.lua:11]: Last time I checked, you should not have tripple dash when it is a continuation of the above .

**Review by piprett — CHANGES_REQUESTED** [review]
There is a lot more I would like to change about the LemmyLua here. There are multiple lies in here. But this is not as urgent as finishing Kristify.
Thanks.

File: src/kristly.lua (hunk 1 above)
Comment (bananasov) [reply src/kristly.lua:11]: Honestly I dont know. If its meant to be expanding the above it should be triple dashes, no?

Comment (bananasov) [reply src/kristly.lua:11]: After looking more into the lua-language-server wiki it does not say anything about this. So i assume its fine?

Comment (bananasov) [reply src/kristly.lua:11]: Perhaps we could put it on a seperate line like so:
```lua
---A basic JSON Post function to the krist api.
---
---Decodes the response from json into a table.
```
and it'll show like: 
image

Comment (piprett) [reply src/kristly.lua:11]: I researched, and the double dashes is a feature of LDoc. They should be replaced with tripple.



> Perhaps we could put it on a seperate line like so:
> 
> ```lua
> ---A basic JSON Post function to the krist api.
> ---
> ---Decodes the response from json into a table.
> ```
> 
> and it'll show like: image

I prefer the current one tbh

Comment (bananasov) [reply src/kristly.lua:11]: Alrighty! feel free to merge

**Review by piprett — APPROVED** [review]

**Pull request merged**